src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -245,6 +245,10 @@ private EntityInfo GetEntityInfo<T>()
                     else
                     {
                         var value = entity.GetString(fieldName);
+                        if (value == null)
+                        {
+                            return null;
```

Comment:
Shouldn't this also change the type of this function to `T?`?

Reply:
ToRecord already has a type `T` return type yeah?

Reply:
We getting the value of the parameter of record constructor, not record itself.